include/eve/memory/aligned_allocator.hpp

```diff
@@ -63,7 +68,14 @@ namespace eve
     value_type *allocate(std::size_t n)
     {
       auto nbelem = align(n * sizeof(value_type), over{alignment()});
-      return static_cast<value_type *>( std::aligned_alloc(alignment(), nbelem) );
+
+      #if defined(SPY_COMPILER_IS_MSVC)
+      void *result = _aligned_malloc(nbelem,alignment());
```

Comment:
[aligned free](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/aligned-free?view=msvc-170) is also a thing

Reply:
Fixed